tests/explore_test.sh

```diff
@@ -5,21 +5,111 @@
 
 function suite
 {
-  suite_addTest "testExplore"
+  suite_addTest "explore_files_under_git_repo_Test"
+  suite_addTest "explore_git_log_Test"
+  suite_addTest "explore_parser_Test"
 }
 
-function testExplore
+function setupExplore()
 {
-  local MSG_OUT="GNU"
-  local LOG_OUT="Initial commit"
+  local -r test_path="tests/.tmp"
```

Comment:
Could we move this variable to outside the function? This way it can also be used in`tearDownExplore()` and `explore_git_log_Test()` without needing to redefine.